lib/schema.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.generateSchemas = exports.Field = exports.Variant = void 0;
+require("reflect-metadata");
+const _1 = require(".");
+const error_1 = require("./error");
+const STRUCT_META_DATA_SYMBOL = '__borsh_struct_metadata__';
+const structMetaDataKey = (constructorName) => {
+    return STRUCT_META_DATA_SYMBOL + constructorName;
+};
+/**
+ *
+ * @param kind 'struct' or 'variant. 'variant' equivalnt to Rust Enum
+ * @returns Schema decorator function for classes
+ */
+exports.Variant = (index) => {
```

Comment:
Can we use lowercase property names? `@variant` and `@field` instead of class-case?